MSAL/src/MSALAccount+Internal.h

```diff
@@ -25,61 +25,55 @@
 //
 //------------------------------------------------------------------------------
 
-#import <Foundation/Foundation.h>
+#import "MSALAccount.h"
 
-@interface MSALUser : NSObject <NSCopying>
+@class MSIDAccountIdentifier;
+@class MSIDAADV2IdTokenClaims;
+@class MSIDClientInfo;
+@class MSIDAccount;
+@class MSALAccountId;
 
-/*!
- The displayable value in UserPrincipleName(UPN) format. Can be nil if not returned from the service.
- */
-@property (readonly) NSString *displayableId;
+@interface MSALAccount ()
 
-/*!
- The given name of the user. Can be nil if not returned by the service.
- */
-@property (readonly) NSString *name;
+@property (nonatomic) MSIDAccountIdentifier *lookupAccountIdentifier;
 
-/*!
- The identity provider of the user authenticated. Can be nil if not returned by the service.
+/* TODO: These properties will be public once we agree on having an account per tenant.
+   For now, will keep them here.
  */
-@property (readonly) NSString *identityProvider;
 
 /*!
- Unique identifier of the user. Can be nil if not returned by the service.
+ Account identifier for the target tenant
  */
-@property (readonly) NSString *uid;
+@property (nonatomic) MSALAccountId *localAccountId;
 
 /*!
- Unique tenant identifier of the user. Can be nil if not returned by the service.
+ The displayable name of the account. Can be nil if not returned by the service.
  */
-@property (readonly) NSString *utid;
+@property (nonatomic) NSString *name;
```

Comment:
lets say displayableName here to make it clear.

Reply:
Renamed displayableId to username to be clear. Name is based on the id_token's claim "name", so I'd prefer to just leave it that way. Please let me know if you disagree :)